### PR TITLE
[33] - Modified pom.xml to append properties files during build and added cz…

### PR DESCRIPTION
**Pull request by anandamideShakyan**
Modified pom.xml to append properties files during build and added cz-duration to application.properties

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.  

# Type of change  

- [x] Bug fix  
- [ ] New feature  
- [ ] Enhancement  
- [ ] Refactoring  
- [ ] Documentation  
- [ ] Other ( please specify )  

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -44,6 +44,10 @@
 		<server-id>wildfly-${environment}</server-id>
 		<server-hostname>{hostname-${environment}}</server-hostname>
 		<server-port>{port-${environment}}</server-port>
+		<wildfly.hostname>localhost</wildfly.hostname>
```
Comment (drtechie) [inline pom.xml:47]: Do we need these?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -180,4 +180,32 @@ quality-Audit-PageSize=5
...
+snomedCTPageSize=10
+
+#call retry count
+callRetryConfiguration=3
+##---------------------------------------------------------------------------------------------------------------
+#avni registration Duration
+avniRegistrationLimit=7
+
+#--------------------------NHM Agent Real Time Data----------------------------
+nhm.agent.real.time.data.url= http://175.101.1.83/apps/utility/alive_api.php
```
Comment (drtechie) [inline src/main/resources/application.properties:204]: Should we be hardcoding this IP?

@devikasuresh20, @ravishanigarapu what's this about?

File: pom.xml (hunk 1 above)
Comment (anandamideShakyan) [reply pom.xml]: Removed.

File: src/main/resources/application.properties (hunk 2 above)
Comment (anandamideShakyan) [reply src/main/resources/application.properties]: Removed the hardcoded ip.

**Pull request merged**